MoltenVK/MoltenVK/Commands/MVKCommandEncodingPool.mm

```diff
@@ -227,16 +229,28 @@ static constexpr uint32_t getRenderpassLoadStoreStateIndex(MVKFormatType type) {
     [_mtlClearColorImageComputePipelineState[0] release];
     [_mtlClearColorImageComputePipelineState[1] release];
     [_mtlClearColorImageComputePipelineState[2] release];
+    [_mtlClearColorImageComputePipelineState[3] release];
+    [_mtlClearColorImageComputePipelineState[4] release];
+    [_mtlClearColorImageComputePipelineState[5] release];
     _mtlClearColorImageComputePipelineState[0] = nil;
     _mtlClearColorImageComputePipelineState[1] = nil;
     _mtlClearColorImageComputePipelineState[2] = nil;
+    _mtlClearColorImageComputePipelineState[3] = nil;
+    _mtlClearColorImageComputePipelineState[4] = nil;
+    _mtlClearColorImageComputePipelineState[5] = nil;
 
 	[_mtlResolveColorImageComputePipelineState[0] release];
 	[_mtlResolveColorImageComputePipelineState[1] release];
 	[_mtlResolveColorImageComputePipelineState[2] release];
+	[_mtlResolveColorImageComputePipelineState[3] release];
+	[_mtlResolveColorImageComputePipelineState[4] release];
+	[_mtlResolveColorImageComputePipelineState[5] release];
 	_mtlResolveColorImageComputePipelineState[0] = nil;
 	_mtlResolveColorImageComputePipelineState[1] = nil;
 	_mtlResolveColorImageComputePipelineState[2] = nil;
+	_mtlResolveColorImageComputePipelineState[3] = nil;
+	_mtlResolveColorImageComputePipelineState[4] = nil;
+	_mtlResolveColorImageComputePipelineState[5] = nil;
```

Comment:
Now that there are so many, would it be simpler to roll this into a loop, like...

```
static constexpr uint32_t kColorImageCnt = 6;
...
for (uint32_t i=0; i < kColorImageCnt; i++) {
    [_mtlClearColorImageComputePipelineState[i] release];
    _mtlClearColorImageComputePipelineState[i] = nil;
    [_mtlResolveColorImageComputePipelineState[i] release];
    _mtlResolveColorImageComputePipelineState[i] = nil;
}
```

and also use `kColorImageCnt` to declare array size too.

Reply:
Fixed!